content/pro/openbb-copilot.md

```diff
@@ -0,0 +1,151 @@
+explore this functionality with in-depth and varied questions.
+
+## Using data from the dashboard
+
+By default, OpenBB Copilot can access the data shown on your currently-active
+dashboard.  This allows OpenBB Copilot to answer questions based on the
+information currently visible.
+
+For example, if you ask for a summary of the latest news about Apple, OpenBB
+Copilot will retrieve the Company News widget data on your dashboard (provided
+the widget has been added to the dashboard) to provide an answer.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/aapl_+news_chat_example.jpg" alt="aapl news openbb copilot" width="80%" height="80%" />
+
+When querying OpenBB Copilot, it is important to understand which data OpenBB
+Copilot used to answer your query. As a result, OpenBB Copilot will always cite the data
+source it referenced when responding to your query. 
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/citation_chat_example.jpg" alt="citations openbb copilot" width="80%" height="80%" />
+
+Since OpenBB Copilot can access data from any widget in the active dashboard, we
+encourage users to experiment with adding different kinds of widgets and
+experimenting with various queries.
+
+For example, OpenBB Copilot is particularly effective at summarizing earnings call transcripts
+from the "Earnings Transcripts" widget.
+
+## Creating widgets from OpenBB Copilot responses
+
+Each time OpenBB Copilot generates a response, it will have an "Create widget from text" or "Create widget from table" button. Clicking this button will create a new widget on your dashboard based on the response.
+
+| ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/text_widget_creation_button.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/text_widget_creation.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/text_widget_created.jpg) |
+|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+
+| ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation_button.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_created.jpg) |
+|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+
+## Querying specific widgets only
```

Comment:
@IgorWounds This section should come before the "Creating widgets" section. Let's group everything together in logical order.

I'm expecting something like:

- Introduction
- How chats work
- Retrieving data
  - From the dashboard
  - Explicitly as context
  - User files
    - (with subsections for directly dragging-and-dropping vs. adding as a widget)
  - Direct retrieval
    - (with subsection for custom endpoints)
  - Searching the Web 
- Citations, Artifacts, and Status Updates
- Creating widgets from OpenBB Copilot outputs
- Prompting Guidelines (<-- link to OpenAI's prompting tips page, or summarize it here)